.eslintrc

```diff
@@ -1,14 +1,17 @@
 {
   "extends": ["@folio/eslint-config-stripes"],
   "parser": "@babel/eslint-parser",
+  "ignorePatterns": ["**/*.mock.js"],
   "rules": {
-    "import/prefer-default-export": "off"
+    "import/prefer-default-export": "off",
+    "react-hooks/exhaustive-deps": "off"

```

Comment:
Sorry; rules may not be customized. All repositories must use the rules provided by [eslint-config-stripes](https://github.com/folio-org/eslint-config-stripes/blob/d6df68638296b6df6fba86329a161785b8230391/index.js). If you disagree with them, open a Jira in ESCONF and start a conversation on Slack#stripes-architecture. 

Test file overrides are acceptable, though still not great.

Reply:
@zburke okay, I'll disable lines then, because we don't want to update deps in all hooks, since it proposing stuff that we don't want to set in deps. About tests, what the problem with them since it mock files ?

Reply:
The lint rules about deps are pretty smart, but sometimes they do get things wrong and need to be turned off. That's fine, completely fine, but then add a comment explaining "We don't need deps X and Y because of reason Z". It's fine to break the rules as long as you can demonstrate you understand why the rule is there and why your circumstances are exceptional. It's not OK to blanket turn off rules that you find inconvenient. 

I agree with you on mocks: we don't want to waste time following strict formatting rules in mocks, though it's nice if those files aren't super sloppy. If you want to loosen the test/mock rules here, fine. If you want to do it globally in eslint-config-stripes where everybody will benefit, even better :)

Reply:
@zburke done!